README.md

```diff
@@ -96,19 +96,19 @@ Includes a setup to easily run any of them.
 —list of awesome projects built with grammY.
 Helpful if you want to see some real-world usage.
 
-### [grammY Telegram Chat](https://t.me/grammyjs)
+### [grammY chat](https://t.me/grammyjs)
 
-—Telegram chat where you can ask any question about grammY or bots in general.
+—The chat where you can ask any question about grammY or bots in general.
 We are also open for feedback, ideas, and contributions!
 
 The Russian community chat can be found [here](https://t.me/grammyjs_ru).
 
-### [grammY News Channel](https://t.me/grammyjs_news)
+### [grammY news](https://t.me/grammyjs_news)
 
-—Telegram channel where updates to grammY and the ecosystem are posted.
+—The channel where updates to grammY and the ecosystem are posted.
 We are also [on Twitter](https://twitter.com/grammy_js).
 
-### [Telegram Bot API Reference](https://core.telegram.org/bots/api)
+### [Telegram Bot API reference](https://core.telegram.org/bots/api)
```

Comment:
Now you're using grammY style for TG stuff. Shouldn't it be title case? Yes, we will then have two different casings of the same word